### PR TITLE
Feature login register modals

### PR DESCRIPTION
**Pull request by saugatdhimal**

### CODE REVIEW

File: src/containers/Login/index.js [hunk 1]
```diff
@@ -1,86 +1,157 @@
...
 import SecureTextInput from '@ra/components/Form/SecureTextInput';
 
 import useRequest from 'hooks/useRequest';
-import {dispatchLogin} from 'utils/dispatch';
+import { dispatchLogin } from 'utils/dispatch';
 
 import logo from 'assets/images/logo-dark.svg';
 
 import styles from './styles.scss';
+import ForgotPasswordModal from 'components/ForgotPasswordModal.js';
```
Comment (ft9dipesh) [inline src/containers/Login/index.js]: The folder name has '.js'?

**Comment by timilsinabishal** [comment]
@saugatdhimal remove the changes made into yarn.lock

File: src/containers/Login/index.js (hunk 1 above)
Comment (ft9dipesh) [inline src/containers/Login/index.js]: Change this import path too!

File: .eslintrc.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
         ],
         "linebreak-style": [
             "error",
-            "unix"
+            "windows"
```
Comment (timilsinabishal) [inline .eslintrc.json]: let's keep this unix

**Review by timilsinabishal — CHANGES_REQUESTED** [review]

File: src/components/EnterCodeModal/index.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import { MdClose } from 'react-icons/md';
```
Comment (timilsinabishal) [inline src/components/EnterCodeModal/index.js]: remove space around parenthesis

**Review by timilsinabishal — CHANGES_REQUESTED** [review]

File: src/components/ForgotPasswordModal/index.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        });
+    }, [onClose, showEnterCodeModal]);
+
+    if (!isVisible) {
+        return null;
+    }
+
+    return (
+        <Modal className={styles.modal}>
+            {console.log(inputData)}
```
Comment (ft9dipesh) [inline src/components/ForgotPasswordModal/index.js]: No console logs here.

**Review by ft9dipesh — COMMENTED** [review]
arrange imports

- first core libraries ('react', 'react-router-dom', ...)
- then npm libraries ('react-icons', ...)
- then components ('Form', 'Button', 'Modal', ...)
- then utils/services
- last absolutes ('./styles.scss', ...)

**Review by timilsinabishal — APPROVED** [review]

**Pull request merged**